Automattic/wp-calypso
# Goals Capture: Update styling to match i1 UI.

#### Proposed Changes

* Updates Goals Capture styling to match i1 UI.

#### Testing Instructions

* Go to `calypso.localhost:3000/setup/goals?siteSlug=test153716255.wordpress.com&flags=signup/goals-step`.
* Check for any layout issues in both desktop & mobile view.

<img width="1503" alt="2022-06-21_12-49-01" src="https://user-images.githubusercontent.com/1287077/174782368-ddd1c385-08c8-4b08-b369-7d64daf2b817.png">
<img width="779" alt="2022-06-21_12-49-08" src="https://user-images.githubusercontent.com/1287077/174782374-c9c984db-f32d-46dd-be28-ea79838da3d3.png">
<img width="500" alt="2022-06-21_12-49-17" src="https://user-images.githubusercontent.com/1287077/174782385-ddec67df-2b1d-431a-9700-633d7c8afd69.png">
